### PR TITLE
Add cheatsheet link to website

### PR DESCRIPTION
**Pull request by jvosloo**

### CODE REVIEW

**Comment by jvosloo** [comment]
@1cg please check if I linked/named the file appropriately... and if so please squash merge to get rid of the interim tries 😅

File: www/index.md [hunk 1]
```diff
@@ -63,6 +63,7 @@ experiences without callback hell
 [Companion of **htmx**](https://htmx.org) |
 [**Comparison** with VanillaJS & jQuery](/comparison) |
 [Read the **docs**](/docs) |
+[Cheatsheet](/hyperscript_cheatsheet_v2021-12-20.pdf)
```
Comment (dz4k) [inline www/index.md:66]: Missing a `|` here

Comment (jvosloo) [reply www/index.md:66]: @dz4k I omitted the | on purpose, cause it looks weird if there's a pip at the end of the row, with the playground link on the new row. I checked it on desktop and mobile and it seemed ok, but I think there will be some small resolutions where the links will probably all stack vertically and then it does look weird without the pipe.

What do you suggest I do here?

Comment (jvosloo) [reply www/index.md:66]: Short of a css media query I mean

**Comment by 1cg** [comment]
link was added by dzk

**Pull request closed without merging**